jinahub/indexers/searcher/compound/FaissPostgresIndexer/README.md

```diff
@@ -34,9 +35,48 @@ with:
 
 The folder needs to contain the data exported from your Indexer. Again, see [docs](https://docs.jina.ai/advanced/experimental/indexers/).
 
-#### Direct import from PSQL
+Check [integration tests](https://github.com/jina-ai/executors/tree/main/tests/integration/psql_dump_reload) for an example on how to use it.
+
+### Direct import from PSQL
+
+Alternatively, you can use the import from PSQL feature. 
+
+This allows you to bypass the expensive writing to disk operation. It also removes the complexities that can arise out of doing this in a cloud environment, where you would need to mount the volume in one instance, and then migrate it, and then perform the import. 
+
+The recommended way to do it is via the **delta import** method. This simply imports the data in your PSQL in each shard, distributing it to each shard. If the data changes during this process, you will have **inconsistency** between the shards. This can manifest in missing results or inconsistency between the data in the Searcher vs data in PSQL. 
+
+```python
+with get_my_flow() as flow:
+    flow.index(your_docs)
+    flow.post(
+        on='/sync',
+        # we need some specific parameters here
+        parameters={
+            'only_delta': True, 'startup': True
+        }
+    )
+    flow.search(search_docs)
+```
+
+However, this can **not** guarantee consistency, but is fast and with low overhead. 
```

Comment:
"While this can **not** guarantee consistency, it is fast and has low overhead."

Sounds less scary.